README.md

```diff
@@ -11,6 +11,11 @@ FLEX (Flipboard Explorer) is a set of in-app debugging and exploration tools for
 <img alt="Demo" width=36% height=36% src=https://user-images.githubusercontent.com/8371943/70185687-e842c800-16af-11ea-8ef9-9e071380a462.gif>
 
 
+## How to generate headermaps?
+- Headermaps are required to build with Bazel
+- Since this repository is not in active development so we are manually checking-in generated header maps. If this repo goes in active development We can setup lanes to generate header maps.
+- At present, headermap files are generated using this rule - https://github.com/bazel-ios/rules_ios/blob/f5fd9581d154647bf92ef76ff65dd5175e7c88f4/rules/hmap.bzl#L97 
```

Comment:
I tried generating using the above rule and then even copying artifacts generated from rules_ios, and  taking it from rules_ios seems good, because while generating on our own we need to do diff and calculate private headers on our own. So In the future to automate this is what I was thinking.

- Create an Example App and add this Pod.
- Build it with Bazel using rules_ios 
- Then copy generated generated hmap files and place it them in root.

Should I update these same steps?

Reply:
I am mostly looking for what the full bazel/bazelisk command one needs to invoke to generate this command?

Let's briefly mention that the sample app needs to have a valid WORKSPACE file with the `rules_ios` rule.

And then devs need to execute this CLI command to get the hmap:

Something like:

`bazelisk build FLEX//:FLEX --generate-hmap`

Reply:
Done. Setup WORKSPACE and BUILD files. also added bazel build command to run in readMe.

- Only copying generated files is a manual step now/